src/featurestorebundle/databricks/feature/writer/DatabricksFeatureStoreCreator.py

```diff
@@ -0,0 +1,52 @@
+from typing import Union, Optional
+from logging import Logger
+from pyspark.sql.types import StructType
+from featurestorebundle.delta.TablePropertiesSetter import TablePropertiesSetter
+from featurestorebundle.databricks.DatabricksFeatureStoreClientFactory import DatabricksFeatureStoreClientFactory
+from featurestorebundle.databricks.feature.writer.DatabricksFeatureStoreExistenceChecker import DatabricksFeatureStoreExistenceChecker
+
+
+class DatabricksFeatureStoreCreator:
+    def __init__(
+        self,
+        logger: Logger,
+        feature_store_client_factory: DatabricksFeatureStoreClientFactory,
+        feature_store_existence_checker: DatabricksFeatureStoreExistenceChecker,
+        table_properties_setter: TablePropertiesSetter,
+    ):
+        self.__logger = logger
+        self.__table_properties_setter = table_properties_setter
+        self.__feature_store_existence_checker = feature_store_existence_checker
+        self.__feature_store_client = feature_store_client_factory.create()
+
+    def create_if_not_exists(self, full_table_name: str, path: str, schema: StructType, partition_by: Optional[Union[str, list]] = None):
```

Comment:
```suggestion
    from typing import Union, Optional, List
    ...
    def create_if_not_exists(self, full_table_name: str, path: str, schema: StructType, partition_by: Optional[Union[str, List[str]]] = None):
```

Reply:
According to the signature in [docs](https://docs.databricks.com/_static/documents/feature-store-python-api-reference-0-5-0.pdf).